CHANGELOG.md

```diff
@@ -0,0 +1,13 @@
+## 0.9.1
+
+* The Broccoli build system was replaced by the Gulp build system
+* Complete replacement of the Backbone models with a Flux structure.
+  The components retrieve their data from stores via events on place.
+  Actions now fetch data from the Marathon API through an replaceable
+  Ajax-wrapper.
+
+## 0.9.0
+
+* Marathon UI lives now in it's own Git repository
+* Update to React 0.13
+* Testing via Mocha
```

Comment:
Could we format the change log so it's easier to read for the community? We could do something like this (only use the necessary sections per release):

# {VERSION} {DATE}

## New Features
...

## Bug Fixes
...

## Breaking Changes
...